tpchgen/src/generators.rs

```diff
@@ -1186,16 +1186,16 @@ impl fmt::Display for PartSupp<'_> {
     }
 }
 
-/// Generator for PartSupplier table data
-#[derive(Debug)]
-pub struct PartSupplierGenerator<'a> {
+/// Generator for PartSupp table data
+#[derive(Debug, Clone)]
+pub struct PartSuppGenerator<'a> {
```

Comment:
I also renamed PartSupplier --> PartSupp to be consistent with TPCH terminology elsewhere